tasks/tasks.py

```diff
@@ -189,10 +189,12 @@ class RunPytest(JobTask):
     action_name = 'run_pytest'
 
     def __init__(self, template, build_url, test_suite, topology=None,
-                 timeout=constants.RUN_PYTEST_TIMEOUT, **kwargs):
+                 timeout=constants.RUN_PYTEST_TIMEOUT, update_packages=False,
+                 **kwargs):
         super(RunPytest, self).__init__(template, timeout=timeout, **kwargs)
         self.build_url = build_url + '/'
         self.test_suite = test_suite
+        self.update_packages = update_packages
```

Comment:
Can we change this to `__update_packages` since it's not used out of the class?

Reply:
I don't see your argument here. Could you provide it in some constructively way? From my POV, there is no need to provide details of the implementation, if the attribute is not part of the API.

Reply:
What's the problem with it being "public" and you being able to change its value in the lifetime of the object?
Your `build_url` and `test_suite` variable are "public", why suddenly change the new variable to be private?

Reply:
That's the point, IMO `build_url` and `test_suite` should be private as well. If objects don't have their state changed, the code is easier to debug, understand and maintain.

Reply:
Is there a reason to hide it? I think not. On the contrary, I could imagine we might want to use the variable in some derived class. I don't think it's Pythonic to hide everything you can and it actually reduces readability.

Even if there was a reason to hide/protect it, I'd go with a single underscore, to just indicate this is something you shouldn't mess with, but to not invoke name mangling. These are used for example for `_run()` and similar methods, because you should use the object as callable instead of invoking these methods manually.

Reply:
> Is there a reason to hide it? I think not.

IMHO, you should have a reason to expose not the opposite. The API should be the simplest possible from the user POV. So, if there are too many attributes/methods/whatever, it's an indicator that your code is not easy to use. Anyway, we can go with one underscore then.

Reply:
IMO the default visibility (public vs _hidden vs __protected) may differ project by project. So, first choose you default and then use it consistently in the project.
I (and this is subjective)  don't like __protecting attributes unless there's reason for it but don't have special feelings for public and _hidden attributes.

Reply:
There is no precedent in using `_` or `__` in attribute names in the `tasks` module.

@felipevolpone If you'd like to change this, please move this discussion and change to a separate PR, which will address **all** the same cases, to be consistent. I found at least:
- `Task.tasks`
- `Task.exc`
- `FallibleTask.raise_on_err`
- `PopenTask.env`
- `PopenTask.process`
- `JobTask.template_name`
- `JobTask.template_version`
- `JobTask.remote_url`
- `Build.git_respec`
- `Build.git_version`
- `Build.git_repo`
- `RunPytest.build_url`
- `RunPytest.test_suite`
- `VagrantBoxDownload.box`
- `VagrantBox.name`
- `VagrantBox.version`
- `VagrantBox.provider`
- `AnsibleFixKeysPermissions.directory`
- `AnsiblePlaybook.extra_vars`
- `AnsiblePlaybook.playbook`
- `AnsiblePlaybook.verbosity`